eudi_wallet_python/satosa/backend.py

```diff
@@ -0,0 +1,163 @@
+        self.redirect_url = self.config['redirect_endpoint']
+        self.request_url = self.config['request_enpoint']
+
+    def register_endpoints(self):
+        """
+        Creates a list of all the endpoints this backend module needs to listen to. In this case
+        it's the authentication response from the underlying OP that is redirected from the OP to
+        the proxy.
+        :rtype: Sequence[(str, Callable[[satosa.context.Context], satosa.response.Response]]
+        :return: A list that can be used to map the request to SATOSA to this endpoint.
+        """
+        url_map = []
+        url_map.append((f"^{self.qrCode_url.lstrip('/')}$", self.qrCode_endpoint))
+        url_map.append((f"^{self.redirect_url.lstrip('/')}$", self.redirect_endpoint))
+        url_map.append((f"^{self.request_url.lstrip('/')}$", self.request_enpoint))
+        return url_map
+
+
+    def qrCode_endpoint(self, context, *args):
+        return Response(
+            text_type(
+                base64.b64encode(bytes('eudiw://authorize?client_id=https://verifier.example.org&request_uri=https://verifier.example.org/request_uri', 'UTF-8'))
+            ).encode("utf-8"), content="text/xml; charset=utf8"
+        )
+    
+    def redirect_endpoint(self, context, *args):
+        return Response(
+            text_type(
+                """
+                {
+                    "request: "ewogICJ0eXAiOiAiZHBvcCtqd3QiLAogICJhbGciOiAiRVMyNTYiLAogICJqd2siOiB7CiAgICAia3R5IjogIkVDIiwKICAgICJ4IjogImw4dEZyaHgtMzR0VjNoUklDUkRZOXpDa0RscEJoRjQyVVFVZldWQVdCRnMiLAogICAgInkiOiAiOVZFNGpmX09rX282NHpiVFRsY3VOSmFqSG10NnY5VERWclUwQ2R2R1JEQSIsCiAgICAiY3J2IjogIlAtMjU2IgogIH0KfQ.ewogICJqdGkiOiAiZjQ3Yzk2YTEtZjkyOC00NzY4LWFhMzAtZWYzMmRjNzhhYTY5IiwKICAiaHRtIjogIkdFVCIsCiAgImh0dSI6ICJodHRwczovL3ZlcmlmaWVyLmV4YW1wbGUub3JnL3JlcXVlc3RfdXJpIiwKICAiaWF0IjogMTU2MjI2MjYxNiwKICAiYXRoIjogImZVSHlPMnIyWjNEWjUzRXNOcldCYjB4V1hvYU55NTlJaUtDQXFrc21RRW8iCn0"
+                }
+                """
+            ).encode("utf-8"), content="text/json; charset=utf8"
+        )
+    
+    def request_enpoint(self, context, *args):
+        return Response(
+            text_type(
+                """
+                {
+                    "response: "ewogICJhbGciOiAiRVMyNTYiLAogICJ0eXAiOiAiSldUIiwKICAia2lkIjogImUwYmJmMmYxLThjM2EtNGVhYi1hOGFjLTJlOGYzNGRiOGE0NyIKfQ.ewogICJpc3MiOiAiaHR0cHM6Ly93YWxsZXQtcHJvdmlkZXIuZXhhbXBsZS5vcmcvaW5zdGFuY2UvdmJlWEprc000NXhwaHRBTm5DaUc2bUN5dVU0amZHTnpvcEd1S3ZvZ2c5YyIsCiAgImp0aSI6ICIzOTc4MzQ0Zi04NTk2LTRjM2EtYTk3OC04ZmNhYmEzOTAzYzUiLAogICJhdWQiOiAiaHR0cHM6Ly92ZXJpZmllci5leGFtcGxlLm9yZy9jYWxsYmFjayIsCiAgImlhdCI6IDE1NDE0OTM3MjQsCiAgImV4cCI6IDE1NzMwMjk3MjMsCiAgIm5vbmNlIjogIm4tMFM2X1d6QTJNaiIsCiAgInZwIjogIjxTRC1KV1Q-fjxEaXNjbG9zdXJlIDE-fjxEaXNjbG9zdXJlIDI-fi4uLn48RGlzY2xvc3VyZSBOPiIKfQ"
+                }
+                """
+            ).encode("utf-8"), content="text/json; charset=utf8"
+        )
+
+    def _metadata_endpoint(self, context):
```

Comment:
this is for SAML2 metadata and it should be remove, we just need to register another endpoint where the entity configuration is published 

`.well-known/openid-federation`

Reply:
Partially resolved in commit 03c6a0bf559097d76770bfcb61a36613718be6bc